src/css/_docs.scss

```diff
@@ -46,11 +46,10 @@
     &-overlay {
       position: relative;
       margin: 0 auto;
-      @include mq(mobile) {
```

Comment:
This is because we still need width: 90% even if it's a weird phone with like a 300px wide screen